tests/tensorflow/data/ac_configs/mask_rcnn_baseline.yml

```diff
@@ -0,0 +1,20 @@
+models:
+  - name: mask_rcnn_baseline
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: mask_rcnn
+          detection_out: reshape_do_2d
+          raw_masks_out: masks
+        inputs:
+          - name: image_info
+            type: CONST_INPUT
+            value: [[800, 1365, 1]]
```

Comment:
@pfinashx, as I know OV does not support MaskRCNN. Do you really need an accuracy config for this model? Also, The AC config is incorrect, at least the input resolution.

Reply:
As I know, support MaskRCNN is planned, so if we take last OV master, maybe it will support these models soon. Now on develop we don't even try to execute it in tests.
I know, that config is wrong, but I did not have time to debug it.
If you have information, that MaskRCNN will not be support, I will delete changes and config.

Reply:
After discussion with @eaidova was found out, that implementation TF mask_rcnn models will last for a long time, so config deleted, skip this tests returned.

Reply:
I guess OV will support it soon. Currently, I see no reason to add the incorrect AC config to the repository. I suggest adding AC config after MO converts the model to test, if it is possible. Will converting model to IR run without AC config?

Reply:
Yes, converting will run, and IR will be collected in artifacts, if converting ends successfully.